provider/awssd/aws_sd.go

```diff
@@ -274,9 +583,20 @@ func (p *AWSSDProvider) submitCreates(namespaces []*sd.NamespaceSummary, changes
 		}
 
 		for _, ch := range changeList {
+			// skip SRV endpoints here. SRV endpoints are handled as part of A endpoints, and registered under an A+SRV service.
+			// skip endpoints with empty DNSName. These are empty endpoints, likley coming from SRV handleSRVChanges
```

Comment:
This comment may be obsolete

Reply:
still needed. I added a missing condition in the if